### ISSUE TITLE
DUPLICATE_TYPE Error when using alongside WPGraphQL Upload

### ISSUE THREAD

**Issue by blackge** [wp-graphql/wp-graphql-buddypress#78]
When I have WPGraphQL Upload activated, I'm getting the following error. If I disable it, the error goes away.

```
  "type": "DUPLICATE_TYPE",
        "message": "You cannot register duplicate Types to the Schema. The Type 'Upload' already exists in the Schema. Make sure to give new Types a unique name.",
        "type_name": "Upload",
```

**Comment by renatonascalves** [comment]
That's tricky. It's likely that the core plugin, wp-graphql, will use this `Upload` type. Since this extension implementation currently doesn't support this plugin upload system, you need to remove it to use it. We don't support multipart upload currently.

But it is on the radar. :)

**Comment by blackge** [comment]
Thanks Renato. I'll have a think about what we need for our site. Do you want to leave this issue open?

**Comment by renatonascalves** [comment]
Yes! It's most likely I'll remove this custom type from this extension and use the one from the wp-graphql plugin when that happens.

**Comment by blackge** [comment]
Good plan!
